tools/setup_perf_cluster.sh

```diff
@@ -238,6 +259,15 @@ function setup_all() {
   setup_cluster_all
 }
 
+function delete_all() {
+  echo "Deleting Istio mesh, cluster $CLUSTER_NAME, Instance $VM_NAME and firewall rules for project $PROJECT in zone $ZONE"
+  sleep 3
```

Comment:
add
echo "Interrupt now if you don't want to delete..."
and sleep 5s instead of 3